pype/modules/ftrack/ftrack_server/lib.py

```diff
@@ -51,9 +51,8 @@ def get_ftrack_event_mongo_info():
 
     if not _used_ftrack_url or components["database"] is None:
         components["database"] = database_name
-    components["collection"] = collection_name
```

Comment:
This line is required (for cases that `AVALON_MONGO` has collection in).

If connection won't work with `collection` in uri then there should be `components.pop("collection", None)` to be sure.

Reply:
Good point. Done.